.github/workflows/run_torchbench.yml

```diff
@@ -39,10 +39,8 @@ jobs:
           # shellcheck source=/dev/null
           . "${SETUP_SCRIPT}"
           conda activate pr-ci
-          # pin cmake version to 3.22 since 3.23 breaks pytorch build
-          # see details at: https://github.com/pytorch/pytorch/issues/74985
           conda install -y numpy="${NUMPY_VERSION}" requests ninja pyyaml mkl mkl-include \
-                           setuptools cmake=3.22 cffi typing_extensions boto3 \
+                           setuptools cmake=3.23.1 cffi typing_extensions boto3 \
```

Comment:
lol, I misread this as `3.22.1` while it's `3.23.1`.  We used to fix the version to the former till recently https://github.com/pytorch/pytorch/pull/90307 where it was relaxed a bit to `3.22.*`